examples/example_led.c

```diff
@@ -0,0 +1,136 @@
+#include <r502.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <limits.h>
+#include <errno.h>
+
+
+int main(int argc, char *argv[]) {
+    Driver *driver;
+    Reply reply;
+    int32_t err = SUCCESS;
+
+    if (argc == 2 && strncmp(argv[1], "--help", 6) == 0) {
+        printf("Usage: %s <port> <control> <speed> <color> <times>\n", argv[0]);
+        printf("port - your port, ex: \"/dev/ttyS0\" if you're using UART on RaspberryPi\n");
```

Comment:
Keep the examples platform and hardware agnostic. Just specify that `port` is the UART communication device.

Reply:
Done!